geopyspark/avroserializer.py

```diff
@@ -10,13 +9,10 @@
 class AvroSerializer(FramedSerializer):
     def __init__(self,
             schema_json,
```

Comment:
We still need that schema. It needs to be decoded by Avro so that we know what the types are in the `RDD`. https://github.com/jbouffard/geopyspark/blob/refactor-3-the-reckoning/geopyspark/avroserializer.py#L31